sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo/Traces/TraceDemo.cs

```diff
@@ -36,7 +36,7 @@ public TraceDemo(string connectionString, TokenCredential? credential = null)
                             .AddSource(ActivitySourceName)
                             .AddProcessor(new ActivityFilteringProcessor())
                             .AddProcessor(new ActivityEnrichingProcessor())
-                            .SetSampler(new ApplicationInsightsSampler(new ApplicationInsightsSamplerOptions() { SamplingRatio = 0.0F }))
+                            .SetSampler(new ApplicationInsightsSampler(new ApplicationInsightsSamplerOptions() { SamplingRatio = 1.0F }))
```

Comment:
may want to do this change as a separate PR.

Reply:
It's accidentally changed as a part of this PR - https://github.com/Azure/azure-sdk-for-net/pull/36111. As it not in the core part of the code, included the change in this PR.